webui/react/src/components/ResourcePoolCardLight.tsx

```diff
@@ -9,20 +9,19 @@ import SlotAllocationBar from 'components/SlotAllocationBar';
 import { V1ResourcePoolTypeToLabel, V1SchedulerTypeToLabel } from 'constants/states';
 import { useStore } from 'contexts/Store';
 import { V1ResourcePoolType, V1SchedulerType } from 'services/api-ts-sdk';
+import * as Api from 'services/api-ts-sdk';
 import { ShirtSize } from 'themes';
-import { ResourcePool, ResourceState, ResourceType } from 'types';
+import { deviceTypes, ResourcePool } from 'types';
+import { getSlotContainerStates } from 'utils/cluster';
 import { clone } from 'utils/data';
 
 import Json from './Json';
-import Link from './Link';
 import css from './ResourcePoolCardLight.module.scss';
-import ResourcePoolDetails from './ResourcePoolDetails';
 
 interface Props {
-  computeContainerStates: ResourceState[];
+  poolStats?: Api.V1RPQueueStat | undefined;
```

Comment:
nitpick / optional - in this file you have `import * as Api from 'services/api-ts-sdk';` but only use it for `V1RPQueueStat` here. In this and the other files with `import * as Api` I'd recommend importing only what you need. It's just for clarity on what depends on what.

Reply:
That surely would be better, it's been fixed.

Reply:
let's chat about which one we prefer to make sure we follow the same style. the team had a chat about this and I think we picked the other style.